components/menu/index.en-US.md

```diff
@@ -0,0 +1,69 @@
+---
+category: Components
+chinese: 导航菜单
+cols: 1
+type: Navigation
+english: Menu
+---
+
+Menu list of Navigation.
+
+## When to use
+
+Navigation menu is important for a website, which makes users jumping from one page to another. Mostly it includes top navigation and side navigation. Top navigation provides all the category and functions of the website. Side navigation provides the Multi-level structure of the website.
```

Comment:
Navigation menu is important for a website, it helps users jump from one site section to another quickly. Mostly, it includes top navigation and side navigation. Top navigation provides all the category and functions of the website. Side navigation provides the Multi-level structure of the website.